mne/commands/mne_kit2fiff.py

```diff
@@ -40,9 +40,15 @@ def run():
     parser.add_option('--output', dest='out_fname',
                       help='Name of the resulting fiff file',
                       metavar='filename')
+    parser.add_option('--debug', dest='debug', action='store_true',
+                      default=False,
+                      help='Set logging level for terminal output to debug')
 
     options, args = parser.parse_args()
 
+    if options.debug:
+        mne.set_log_level('debug')
```

Comment:
Does it do that? From what I can tell it just calls `logger.setLevel()` (I also called `$ mne kit2fiff` repeatedly and the log level did not seem to get stuck): https://github.com/mne-tools/mne-python/blob/master/mne/utils.py#L1239